github/ci/prow/files/jobs/kubevirt/kubevirtci/kubevirtci-periodics.yaml

```diff
@@ -0,0 +1,52 @@
+periodics:
+- name: periodic-kubevirtci-sync-crio-repository-weekly
+  interval: 168h
+  decorate: true
+  decoration_config:
+    timeout: 1h
+    grace_period: 5m
+  max_concurrency: 1
+  labels:
+    preset-docker-mirror-proxy: "true"
+    preset-shared-images: "true"
+  spec:
+    containers:
+      - image: kubevirtci/bootstrap:v20201119-a5880e0
+        env:
+          - name: GOOGLE_APPLICATION_CREDENTIALS
+            value: /etc/gcs/service-account.json
+        command:
+          - "/usr/local/bin/runner.sh"
+          - "/bin/sh"
+          - "-c"
+          - >
+            set -e;
+            BUCKET=kubevirtci-crio-mirror
+            LOCAL_MIRROR_DIR=opensuse-mirror
+            OS=CentOS_8_Stream
+            CRIO_VERSION=1.20
```

Comment:
I think it would be better to define these env vars in the `env` block, wdyt?

Reply:
Done